tests/test_issues.py

```diff
@@ -253,3 +252,19 @@ def test_issue_77_async_generator_partial():
     assert inspect.isasyncgenfunction(f_partial)
 
     assert asyncio.get_event_loop().run_until_complete(asyncio.ensure_future(f_partial().__anext__())) == 1
+
+
+def test_issue_85_wrapped_forwardref_annotation():
+    import typing
+    from . import issue_85_module
+
+    @wraps(issue_85_module.forwardref_method, remove_args=["bar"])
+    def decorated(*args, **kwargs):
```

Comment:
Note that I would typically not put args here as they are always empty (makefun passes all as keyword except positional-only)

Reply:
Done. I also changed the syntax to something python 2 can at least parse.